contracts/script/deploy/mainnet/mainnet.beta.config.toml

```diff
@@ -0,0 +1,89 @@
+### CORE ###
+
+# This address gets all privileges at the end of the deployment.
+initialOwner = "0x002721B4790d97dC140a049936aA710152Ba92D5"
+
+# Parameters shared across various deployed contracts
+[initParams.shared]
+rewardsCoordinator = "0x7750d328b314EfFa365A0402CcfD489B80B0adda"
+avsDirectory = "0x135DDa560e946695d6f155dACaFC6f1F25C1F5AF"
+delegationManager = "0x39053D51B77DC0d36036Fc1fCc8Cb819df8Ef37A"
+initialPausedStatus = 0
+
+# Parameters for the pauser registry contract
+[initParams.core.pauserRegistry]
+pausers = ["0x002721B4790d97dC140a049936aA710152Ba92D5"]
+unpauser = "0x002721B4790d97dC140a049936aA710152Ba92D5"
+
+### MIDDLEWARE ###
+
+# Parameters for the registry coordinator contract. Copied from mainnet.
+[initParams.middleware.registryCoordinator]
+churnApprover = "0xe0550117Cb066D3b330eBd764B0d75D3BA378734"
+minimumStakes = ["32000000000000000000", "1000000000000000000", "1000000000000000000"] # Strings for toml address parser compatibility reasons
+strategyParams = [
+    [
+        { 0_strategy = "0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0", 1_multiplier = 1000000000000000000 },
+        { 0_strategy = "0x93c4b944D05dfe6df7645A86cd2206016c51564D", 1_multiplier = 1043185676128837999 },
+        { 0_strategy = "0x1BeE69b7dFFfA4E2d53C2a2Df135C388AD25dCD2", 1_multiplier = 1114663583060673944 },
+        { 0_strategy = "0x54945180dB7943c0ed0FEE7EdaB2Bd24620256bc", 1_multiplier = 1080022650414740066 },
+        { 0_strategy = "0x9d7eD45EE2E8FC5482fa2428f15C971e6369011d", 1_multiplier = 1038703328428972081 },
+        { 0_strategy = "0x13760F50a9d7377e4F20CB8CF9e4c26586c658ff", 1_multiplier = 1167295905003755853 },
+        { 0_strategy = "0xa4C637e0F704745D182e4D38cAb7E7485321d059", 1_multiplier = 1027044953080930383 },
+        { 0_strategy = "0x57ba429517c3473B6d34CA9aCd56c0e735b94c02", 1_multiplier = 1025010945212823010 },
+        { 0_strategy = "0x0Fe4F44beE93503346A3Ac9EE5A26b130a5796d6", 1_multiplier = 1068966896363604679 },
+        { 0_strategy = "0x7CA911E83dabf90C90dD3De5411a10F1A6112184", 1_multiplier = 1047995874333000000 },
+        { 0_strategy = "0x8CA7A5d6f3acd3A7A8bC468a8CD0FB14B6BD28b6", 1_multiplier = 1096547124777235201 },
+        { 0_strategy = "0xAe60d8180437b5C34bB956822ac2710972584473", 1_multiplier = 1057040013302350278 },
```

Comment:
what are these addresses? Do we have a reference? How are the multipliers chosen? I'm guessing you copied them from somewhere else? Please add a link

Reply:
Copied from mainnet

Reply:
Who knows what they are and how they were chosen? Only Arch? Can we find who has this knowledge and add comments here? This is our main source of information right? Or is this already documented elsewhere?

Reply:
I was not around for choosing these parameters, so I personally have only a surface knowledge. This was supposedly the list of all staked assets on eigenlayer at the time. I have no idea how the multipliers were chosen.